templates/app/middlewares/forestadmin.txt

```diff
@@ -6,8 +6,8 @@ const mongoose = require('mongoose');<% } else { %>
 const Liana = require('forest-express-sequelize');
 const { sequelize } = require('../models');<% } %>
 
-module.exports = function (app) {
-  app.use(Liana.init({
+module.exports = async function (app) {
+  app.use(await Liana.init({
```

Comment:
<img width="730" alt="Screenshot 2019-12-02 at 17 43 04" src="https://user-images.githubusercontent.com/3200482/69977523-41352380-152b-11ea-8d0d-764a8de22797.png">
`Liana.init` now returns a promise. Here, we must await Liana.init to avoid passing a promise to express.